### ISSUE TITLE
Rename the tool

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wai-wcag-em-report-tool#63]
Something along the lines of "WCAG-EM report Generator" or "WCAG Evaluation Assistant". The current name is confusing users about its purpose. This issue should be worked on together with the Education & Outreach Working Group.

**Comment by hidde** [comment]
@nitedog 

Joe Lanman [said on Twitter](https://twitter.com/joelanman/status/1428312325543632900?s=20):

> is it enough to differentiate them as ATAG Report Tool and WCAG Report Tool? What is the EM part adding in terms of name?

We could consider using WCAG Report Tool and removing `-EM` from the name. 

Pro: reduces name to essence; is even more consistent with ATAG Report Tool name; works with current URL
Con: most current users will be used to the old name; the tool is specifically about the EM 

Probably be best to do this before shipping 3.0.0 as that seems like a good timing to do a name change.

**Comment by nitedog** [comment]
I'm not opposed to this but WCAG-EM Report Tool reflects the [WCAG-EM specification](https://www.w3.org/TR/WCAG-EM/), not really WCAG itself. That is, I can see a counter-argument here. According to the [EOWG Process for Minor Changes](https://www.w3.org/WAI/EO/wiki/EOWG_Participation_Info#Process_for_Minor_Changes), the chairs and staff contact need to determine if this is a minor change or if it requires further group review and approval. Please bring it to their attention.

**Comment by hidde** [comment]
Will bring to chairs' attention in next update, thanks Shadi.

From my perspective it is both a WCAG Report Tool (reporting on WCAG evaluation) and a WCAG-EM Report Tool (reporting on WCAG evaluation that followed the EM): as in, 'WCAG-EM' is the more specific and 'WCAG' the more generic.

**Comment by SteveALee** [comment]
If the aim is to provide a tool for generating a report then I feel most users will not neccessarily know or vare about the exact format. So I would drop the EM from the name, but keep the reference in the introductory text for those who do know or care about the EM format.
